scripts/code_style.py

```diff
@@ -209,10 +217,10 @@ def main() -> int:
 
     if args.fix:
         # Fix mode
-        return fix_style(src_files)
+        return fix_style(args.uncrustify, src_files)
```

Comment:
The [default ](https://github.com/Mbed-TLS/mbedtls/blob/03d003f3018a2ca1669be5604c00da691d1e7e7a/scripts/code_style.py#L192) for `args.uncrustify` could be set to be to `UNCRUSTIFY_EXE` set at at [line17](https://github.com/Mbed-TLS/mbedtls/blob/03d003f3018a2ca1669be5604c00da691d1e7e7a/scripts/code_style.py#L17).

Reply:
`UNCRUSTIFY_EXE` shouldn't exist anymore... But I missed an occurrence. I'll fix that.